docs/readthedocs/about.md

```diff
@@ -1,10 +1,20 @@
 (about)=
-# About SIMONA
+# About Simona
 
-## Overview
+Simona is a simulation-tool for generating time-series in the context of distribution grids to be used in grid planning, operation and analysis. The simulation is executed using an agent-based, discrete-event model. Simona is being developed at the [Institute of Energy Systems, Energy Efficiency and Energy Economics](https://ie3.etit.tu-dortmund.de) at [TU Dortmund University](https://www.tu-dortmund.de), Germany.
```

Comment:
I would go for SIMONA in capital letters and I think it is simulation tool as well as time-series as separate words.